.github/renovate.json

```
@@ -22,33 +40,70 @@
         "^eslint$"
       ]
     },
+    {
+      "enabled": true,
+      "groupName": "GraphQL",
+      "matchManagers": ["npm"],
+      "matchPackagePatterns": ["^@graphql/", "^graphql$", "^graphql-"]
+    },
     {
       "enabled": true,
       "groupName": "Jest",
       "matchManagers": ["npm"],
-      "matchPackagePatterns": ["^@types/jest$", "^jest", "^ts-jest$", "^jest-"]
+      "matchPackagePatterns": ["^@types/jest$", "jest", "^jest-", "-jest$"]
```

What was the reason for this change? `"jset"` is a very generic pattern (which will actually cover all the other patterns in this line as well).

---

I was hoping to avoid potential false positives, but am realizing that the logic here could be simplified in any case.

And actually, upon further reflection, if we end up with false positives we can just refine it then. Thanks for flagging. Amended in https://github.com/hashintel/hash/commit/198b1c5e5b7dcecfb70d12e7a06d4661d1850708

---

I'm pretty sure the amended version is not what we want. With this we won't capture `@types/jest`, `ts-jest`, `eslint-plugin-jest`, `babel-jest`, `jest-junit`, `jest-message-util` (various others).
```suggestion
"matchPackagePatterns": ["^@types/jest$", "^jest-", "-jest$"]
```

---

My bad. Actually fixed, not broken more, in https://github.com/hashintel/hash/pull/2887/commits/cb8bc7a73ef70dc0cedfdde7071cd6e2413cbaf0